thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs

```diff
@@ -0,0 +1,5 @@
+<p>
+  {{#each-in record.dimensions as |dimName dimList|}}
+    {{dimName}}: {{dimList}}<br>
```

Comment:
I'm personally not a fan of <br> because it's not really semantic html. 
Can we achieve this with css?